README.md

```diff
@@ -201,6 +201,22 @@ or if using dockerized Devnet:
 $ docker run -e RUST_LOG=<LEVEL> shardlabs/starknet-devnet-rs
 ```
 
+By default, logging of request and response data is turned off.
+To see the request or/and response body, additional level have to be provided via `RUST_LOG` environment variable.
+To log the request body use `REQUEST`, to log the response body use `RESPONSE`.
+
+NOTE! that logging request and response requires at least logging level `INFO`.
+
+Example:
+
+```
+$ RUST_LOG="REQUEST, RESPONSE" starknet-devnet
```

Comment:
Was this comma-separation supported so far? Which part of the code provides this functionality? I don't see any explicit splitting

Reply:
its supported by tracing_subscriber library